File: Dermatoloska_ordinacija/UI/dermatoloska_desktop/lib/models/zdravstveni_karton.dart

import 'package:json_annotation/json_annotation.dart';
import 'korisnik.dart';

part 'zdravstveni_karton.g.dart';

@JsonSerializable()

class ZdravstveniKarton {
  int? zdravstveniKartonId;
  String? sadrzaj;
  int? korisnikId;
  Korisnik? korisnik;

  ZdravstveniKarton(this.zdravstveniKartonId, this.sadrzaj, this.korisnikId, this.korisnik);

  factory ZdravstveniKarton.fromJson(Map<String, dynamic> json) => _$ZdravstveniKartonFromJson(json);

  Map<String, dynamic> toJson() => _$ZdravstveniKartonToJson(this);
}

File: DermOrdDodatni/DermOrdDodatni/Database/ZdravstveniKarton.cs
using System;
using System.Collections.Generic;

namespace DermOrdDodatni.Database;


public partial class ZdravstveniKarton
{
    public int ZdravstveniKartonId { get; set; }

    public string? Sadrzaj { get; set; }

    public int? KorisnikId { get; set; }

    public virtual Korisnik? Korisnik { get; set; }
}